nix/deploy/oci-images.nix

```diff
@@ -0,0 +1,32 @@
+{ repoRoot, inputs, pkgs, lib, system }:
+
```

Comment:
I wrote this file thinking that we were going to do docker images the "old way".
Please confirm whether I should delete it.

Reply:
Yes, delete